summary_workflows/identification/I5_benchmark/I5_benchmark_identification.md

```diff
@@ -0,0 +1,105 @@
+# I5 benchmark specification
+
+## Synopsis
+
+Benchmark to compare RNAseq-based site identification with ground truth, i.e. the poly(A) sites of transcripts used to generate simulated RNA-seq data, for several different distance thresholds.
+
+Input data:
+
+1. Poly(A) sites identified based on RNAseq data using the benchmarked tool
+2. Poly(A) sites of transcripts used to generate the simulated RNAseq reads
+
+Based on the input data the following metrics are computed:
+
+1. Precision = (TP/(TP+FP))
+
+TP - true positives - PAS identified by the tool and present within X nucleotides of PAS identified in the database  
+FP - false positives  - PAS identified by the tool and not present within X nucleotides of PAS identified in the database
+
+The metrics should be computed for different distance thresholds between PAS identified by the tool from RNAseq dataset and PAS identified in the database i.e. the PAS identified by the tool should be within X nucleotides from the PAS identified in the database for the prediction to be considered true.  
+The following distance thresholds should be used:
+
+- 0 nt
+- 20 nt
+- 40 nt
+- 60 nt
+- 80 nt
+- 100 nt
+- 120 nt
+- 140 nt
+- 160 nt
+- 180 nt
+- 200 nt
+
+
+## General info
+
+* **Challenge:** Identification
+* **Identifier:** I5
+
+## Inputs
+
+| # | Format | Link | Example data |
+  | --- | --- | --- | --- |
+  | 1 | BED | [Specification][spec-bed] | [Link][in1] |
+  | 2 | BED | [Specification][spec-bed] | [Link][in2] |
+
+### Additional info inputs
+  
+#### Format 1
+
+This BED file contains PAS identified by the benchmarked tool from RNAseq data. Poly(A) sites should be single nucleotide.
+
+#### Format 2
+
+This BED file contains PAS of transcripts used to simulate the RNA-seq data
+
+## Outputs
+
+| # | Format | Link | Example data |
+  | --- | --- | --- | --- |
+  | 1 | JSON | [Specification][spec-json] | [Link][out1] |
+  
+### Additional info outputs
+
+#### Format 1
+  
+  The following table lists the attribute names, value types and units, and a
+description of each attribute-value pair:
+  
+  | Attribute | Type | Unit | Description |
+  | --- | --- | --- | --- |
+  | `Precision` | `vector` | N/A | A vector of length=11 containing precision values of PAS identification compared with PolyASite database; Precision = (TP/(TP+FP)); calculated for distance between 0 nt and 200 nt with 20 nt intervals; Each value in the vector is of type `float` |
```

Comment:
> compared with **PolyASite database**

should be changed to e.g. "ground truth PAS from transcripts used to generate simulated data"